src/WorkflowManager/Logging/Logging/Log.cs

```diff
@@ -114,7 +116,45 @@ public static void TaskComplete(this ILogger logger, TaskExecution task, Workflo
 
             var jsonString = JsonConvert.SerializeObject(objectLog);
 
-            logger.LogInformation(27, message: jsonString);
+            logger.LogInformation(28, message: jsonString);
+        }
+
+        public static void LogControllerStartTime(this ILogger logger, ActionExecutingContext context)
+        {
+            var request = context.HttpContext.Request;
+            var body = context.ActionArguments.FirstOrDefault();
+            var objectLog = new ObjectLog
+            {
+                Message = "ControllerActionStart",
+                Object = LoggerHelpers.ToLogControllerStartObject(request.Method, request.Path, request.QueryString.Value.ToString(), body.Value ?? "", "", "")
+            };
+
+            var jsonString = JsonConvert.SerializeObject(objectLog);
+
+            logger.LogInformation(29, message: jsonString);
+        }
+
+        public static void LogControllerEndTime(this ILogger logger, ResultExecutedContext context)
+        {
+            var request = context.HttpContext.Request;
+            var response = context.HttpContext.Response;
+
+            var objResult = new ObjectResult("");
+
+            if (context.Result is ObjectResult)
```

Comment:
you can do a 
```
ObjectResult objResult;
if (context.Result is ObjectResult as contextObjResult)
{
    objResult = contextObjResult;
}
```

I think is what scanner is saying here.

Reply:
Tried that initially, unfortunately this results in the compilation error "Use of unassigned local variable"